src/test/java/seedu/address/model/person/AttendanceTest.java

```diff
@@ -61,6 +61,15 @@ public void test_attendanceCheckString() {
         assertEquals(testAtt.attendanceCheckString(), " [Absent]");
     }
 
+    @Test
+    public void test_setIsPresent() {
+        Attendance testAtt = new Attendance("2022-08-12");
+        testAtt.setIsPresent(true);
```

Comment:
Can put the attendance string as ATTENDANCE_STUB field for standardisation with other test classes